tests/schemas/movies.esdl

```diff
@@ -1,7 +1,7 @@
 #
 # This source file is part of the EdgeDB open source project.
 #
-# Copyright 2016-present MagicStack Inc. and the EdgeDB authors.
```

Comment:
Go home `git diff`, you are drunk.

(I think it was such a small file that the copyright notice made it think it was the same.)

Reply:
Well, it is doing what it is supposed to. The problem is that the majority of the file is copyright.